CONFIG.md

```diff
@@ -484,6 +484,27 @@ should be placed. Defaults to the package root. For Acquia set this to
 **packages.dest.devResources**: Specify where within the package directory the
 `projFiles` should be placed. Defaults to package root.
 
+### Install settings
+
+The `install` task will either install Drupal via the configured `profile`, or if
+a path to a database is configured, load this database:
+
+To specify a profile other than the `standard` profile:
+
+```json
+{
+  "serve": {
```

Comment:
I'm thinking for B/C we should keep that for now, but maybe a top-level "project" key makes sense?